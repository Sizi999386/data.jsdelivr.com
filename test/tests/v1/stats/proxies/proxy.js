const { makeEndpointSnapshotTests, setupSnapshots } = require('../../../../utils');

const periodOptions = [ 'day', 'week', 'month', 'year', 'all' ];

describe('/v1/stats/proxies/proxy', () => {
	before(() => {
		setupSnapshots(__filename);
	});

	makeProxyStatsTests();
});

function makeProxyStatsTests () {
	let defaults = {
		period: 'month',
	};

	makeEndpointSnapshotTests('/v1/stats/proxies/{name}{?type,period}', defaults, [
		{ name: 'wp-plugins', type: [ 'hits', 'bandwidth' ], period: periodOptions },
	]);
}
