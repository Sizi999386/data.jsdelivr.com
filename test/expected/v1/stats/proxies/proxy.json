{
	"/v1/stats/proxies/wp-plugins?period=all": {
		"date": "2010-12-31",
		"hits": {
			"total": 2346,
			"dates": {
				"2010-10-22": 0,
				"2010-10-23": 1,
				"2010-10-24": 2,
				"2010-10-25": 3,
				"2010-10-26": 4,
				"2010-10-27": 5,
				"2010-10-28": 6,
				"2010-10-29": 7,
				"2010-10-30": 8,
				"2010-10-31": 9,
				"2010-11-01": 10,
				"2010-11-02": 11,
				"2010-11-03": 12,
				"2010-11-04": 13,
				"2010-11-05": 14,
				"2010-11-06": 15,
				"2010-11-07": 16,
				"2010-11-08": 17,
				"2010-11-09": 18,
				"2010-11-10": 19,
				"2010-11-11": 20,
				"2010-11-12": 21,
				"2010-11-13": 22,
				"2010-11-14": 23,
				"2010-11-15": 24,
				"2010-11-16": 25,
				"2010-11-17": 26,
				"2010-11-18": 27,
				"2010-11-19": 28,
				"2010-11-20": 29,
				"2010-11-21": 30,
				"2010-11-22": 31,
				"2010-11-23": 32,
				"2010-11-24": 33,
				"2010-11-25": 34,
				"2010-11-26": 35,
				"2010-11-27": 36,
				"2010-11-28": 37,
				"2010-11-29": 38,
				"2010-11-30": 39,
				"2010-12-01": 40,
				"2010-12-02": 41,
				"2010-12-03": 42,
				"2010-12-04": 43,
				"2010-12-05": 44,
				"2010-12-06": 45,
				"2010-12-07": 46,
				"2010-12-08": 47,
				"2010-12-09": 48,
				"2010-12-10": 49,
				"2010-12-11": 50,
				"2010-12-12": 51,
				"2010-12-13": 52,
				"2010-12-14": 53,
				"2010-12-15": 54,
				"2010-12-16": 55,
				"2010-12-17": 56,
				"2010-12-18": 57,
				"2010-12-19": 58,
				"2010-12-20": 59,
				"2010-12-21": 60,
				"2010-12-22": 61,
				"2010-12-23": 62,
				"2010-12-24": 63,
				"2010-12-25": 64,
				"2010-12-26": 65,
				"2010-12-27": 66,
				"2010-12-28": 67,
				"2010-12-29": 68
			},
			"prev": {
				"total": 0
			}
		},
		"bandwidth": {
			"total": 38474400,
			"dates": {
				"2010-10-22": 0,
				"2010-10-23": 16400,
				"2010-10-24": 32800,
				"2010-10-25": 49200,
				"2010-10-26": 65600,
				"2010-10-27": 82000,
				"2010-10-28": 98400,
				"2010-10-29": 114800,
				"2010-10-30": 131200,
				"2010-10-31": 147600,
				"2010-11-01": 164000,
				"2010-11-02": 180400,
				"2010-11-03": 196800,
				"2010-11-04": 213200,
				"2010-11-05": 229600,
				"2010-11-06": 246000,
				"2010-11-07": 262400,
				"2010-11-08": 278800,
				"2010-11-09": 295200,
				"2010-11-10": 311600,
				"2010-11-11": 328000,
				"2010-11-12": 344400,
				"2010-11-13": 360800,
				"2010-11-14": 377200,
				"2010-11-15": 393600,
				"2010-11-16": 410000,
				"2010-11-17": 426400,
				"2010-11-18": 442800,
				"2010-11-19": 459200,
				"2010-11-20": 475600,
				"2010-11-21": 492000,
				"2010-11-22": 508400,
				"2010-11-23": 524800,
				"2010-11-24": 541200,
				"2010-11-25": 557600,
				"2010-11-26": 574000,
				"2010-11-27": 590400,
				"2010-11-28": 606800,
				"2010-11-29": 623200,
				"2010-11-30": 639600,
				"2010-12-01": 656000,
				"2010-12-02": 672400,
				"2010-12-03": 688800,
				"2010-12-04": 705200,
				"2010-12-05": 721600,
				"2010-12-06": 738000,
				"2010-12-07": 754400,
				"2010-12-08": 770800,
				"2010-12-09": 787200,
				"2010-12-10": 803600,
				"2010-12-11": 820000,
				"2010-12-12": 836400,
				"2010-12-13": 852800,
				"2010-12-14": 869200,
				"2010-12-15": 885600,
				"2010-12-16": 902000,
				"2010-12-17": 918400,
				"2010-12-18": 934800,
				"2010-12-19": 951200,
				"2010-12-20": 967600,
				"2010-12-21": 984000,
				"2010-12-22": 1000400,
				"2010-12-23": 1016800,
				"2010-12-24": 1033200,
				"2010-12-25": 1049600,
				"2010-12-26": 1066000,
				"2010-12-27": 1082400,
				"2010-12-28": 1098800,
				"2010-12-29": 1115200
			},
			"prev": {
				"total": 0
			}
		}
	},
	"/v1/stats/proxies/wp-plugins?period=day": {
		"date": "2010-12-31",
		"hits": {
			"total": 68,
			"dates": {
				"2010-12-29": 68
			},
			"prev": {
				"total": 67
			}
		},
		"bandwidth": {
			"total": 1115200,
			"dates": {
				"2010-12-29": 1115200
			},
			"prev": {
				"total": 1098800
			}
		}
	},
	"/v1/stats/proxies/wp-plugins?period=month": {
		"date": "2010-12-31",
		"hits": {
			"total": 1605,
			"dates": {
				"2010-11-30": 39,
				"2010-12-01": 40,
				"2010-12-02": 41,
				"2010-12-03": 42,
				"2010-12-04": 43,
				"2010-12-05": 44,
				"2010-12-06": 45,
				"2010-12-07": 46,
				"2010-12-08": 47,
				"2010-12-09": 48,
				"2010-12-10": 49,
				"2010-12-11": 50,
				"2010-12-12": 51,
				"2010-12-13": 52,
				"2010-12-14": 53,
				"2010-12-15": 54,
				"2010-12-16": 55,
				"2010-12-17": 56,
				"2010-12-18": 57,
				"2010-12-19": 58,
				"2010-12-20": 59,
				"2010-12-21": 60,
				"2010-12-22": 61,
				"2010-12-23": 62,
				"2010-12-24": 63,
				"2010-12-25": 64,
				"2010-12-26": 65,
				"2010-12-27": 66,
				"2010-12-28": 67,
				"2010-12-29": 68
			},
			"prev": {
				"total": 705
			}
		},
		"bandwidth": {
			"total": 26322000,
			"dates": {
				"2010-11-30": 639600,
				"2010-12-01": 656000,
				"2010-12-02": 672400,
				"2010-12-03": 688800,
				"2010-12-04": 705200,
				"2010-12-05": 721600,
				"2010-12-06": 738000,
				"2010-12-07": 754400,
				"2010-12-08": 770800,
				"2010-12-09": 787200,
				"2010-12-10": 803600,
				"2010-12-11": 820000,
				"2010-12-12": 836400,
				"2010-12-13": 852800,
				"2010-12-14": 869200,
				"2010-12-15": 885600,
				"2010-12-16": 902000,
				"2010-12-17": 918400,
				"2010-12-18": 934800,
				"2010-12-19": 951200,
				"2010-12-20": 967600,
				"2010-12-21": 984000,
				"2010-12-22": 1000400,
				"2010-12-23": 1016800,
				"2010-12-24": 1033200,
				"2010-12-25": 1049600,
				"2010-12-26": 1066000,
				"2010-12-27": 1082400,
				"2010-12-28": 1098800,
				"2010-12-29": 1115200
			},
			"prev": {
				"total": 11562000
			}
		}
	},
	"/v1/stats/proxies/wp-plugins?period=week": {
		"date": "2010-12-31",
		"hits": {
			"total": 455,
			"dates": {
				"2010-12-23": 62,
				"2010-12-24": 63,
				"2010-12-25": 64,
				"2010-12-26": 65,
				"2010-12-27": 66,
				"2010-12-28": 67,
				"2010-12-29": 68
			},
			"prev": {
				"total": 406
			}
		},
		"bandwidth": {
			"total": 7462000,
			"dates": {
				"2010-12-23": 1016800,
				"2010-12-24": 1033200,
				"2010-12-25": 1049600,
				"2010-12-26": 1066000,
				"2010-12-27": 1082400,
				"2010-12-28": 1098800,
				"2010-12-29": 1115200
			},
			"prev": {
				"total": 6658400
			}
		}
	},
	"/v1/stats/proxies/wp-plugins?period=year": {
		"date": "2010-12-31",
		"hits": {
			"total": 2346,
			"dates": {
				"2009-12-30": 0,
				"2009-12-31": 0,
				"2010-01-01": 0,
				"2010-01-02": 0,
				"2010-01-03": 0,
				"2010-01-04": 0,
				"2010-01-05": 0,
				"2010-01-06": 0,
				"2010-01-07": 0,
				"2010-01-08": 0,
				"2010-01-09": 0,
				"2010-01-10": 0,
				"2010-01-11": 0,
				"2010-01-12": 0,
				"2010-01-13": 0,
				"2010-01-14": 0,
				"2010-01-15": 0,
				"2010-01-16": 0,
				"2010-01-17": 0,
				"2010-01-18": 0,
				"2010-01-19": 0,
				"2010-01-20": 0,
				"2010-01-21": 0,
				"2010-01-22": 0,
				"2010-01-23": 0,
				"2010-01-24": 0,
				"2010-01-25": 0,
				"2010-01-26": 0,
				"2010-01-27": 0,
				"2010-01-28": 0,
				"2010-01-29": 0,
				"2010-01-30": 0,
				"2010-01-31": 0,
				"2010-02-01": 0,
				"2010-02-02": 0,
				"2010-02-03": 0,
				"2010-02-04": 0,
				"2010-02-05": 0,
				"2010-02-06": 0,
				"2010-02-07": 0,
				"2010-02-08": 0,
				"2010-02-09": 0,
				"2010-02-10": 0,
				"2010-02-11": 0,
				"2010-02-12": 0,
				"2010-02-13": 0,
				"2010-02-14": 0,
				"2010-02-15": 0,
				"2010-02-16": 0,
				"2010-02-17": 0,
				"2010-02-18": 0,
				"2010-02-19": 0,
				"2010-02-20": 0,
				"2010-02-21": 0,
				"2010-02-22": 0,
				"2010-02-23": 0,
				"2010-02-24": 0,
				"2010-02-25": 0,
				"2010-02-26": 0,
				"2010-02-27": 0,
				"2010-02-28": 0,
				"2010-03-01": 0,
				"2010-03-02": 0,
				"2010-03-03": 0,
				"2010-03-04": 0,
				"2010-03-05": 0,
				"2010-03-06": 0,
				"2010-03-07": 0,
				"2010-03-08": 0,
				"2010-03-09": 0,
				"2010-03-10": 0,
				"2010-03-11": 0,
				"2010-03-12": 0,
				"2010-03-13": 0,
				"2010-03-14": 0,
				"2010-03-15": 0,
				"2010-03-16": 0,
				"2010-03-17": 0,
				"2010-03-18": 0,
				"2010-03-19": 0,
				"2010-03-20": 0,
				"2010-03-21": 0,
				"2010-03-22": 0,
				"2010-03-23": 0,
				"2010-03-24": 0,
				"2010-03-25": 0,
				"2010-03-26": 0,
				"2010-03-27": 0,
				"2010-03-28": 0,
				"2010-03-29": 0,
				"2010-03-30": 0,
				"2010-03-31": 0,
				"2010-04-01": 0,
				"2010-04-02": 0,
				"2010-04-03": 0,
				"2010-04-04": 0,
				"2010-04-05": 0,
				"2010-04-06": 0,
				"2010-04-07": 0,
				"2010-04-08": 0,
				"2010-04-09": 0,
				"2010-04-10": 0,
				"2010-04-11": 0,
				"2010-04-12": 0,
				"2010-04-13": 0,
				"2010-04-14": 0,
				"2010-04-15": 0,
				"2010-04-16": 0,
				"2010-04-17": 0,
				"2010-04-18": 0,
				"2010-04-19": 0,
				"2010-04-20": 0,
				"2010-04-21": 0,
				"2010-04-22": 0,
				"2010-04-23": 0,
				"2010-04-24": 0,
				"2010-04-25": 0,
				"2010-04-26": 0,
				"2010-04-27": 0,
				"2010-04-28": 0,
				"2010-04-29": 0,
				"2010-04-30": 0,
				"2010-05-01": 0,
				"2010-05-02": 0,
				"2010-05-03": 0,
				"2010-05-04": 0,
				"2010-05-05": 0,
				"2010-05-06": 0,
				"2010-05-07": 0,
				"2010-05-08": 0,
				"2010-05-09": 0,
				"2010-05-10": 0,
				"2010-05-11": 0,
				"2010-05-12": 0,
				"2010-05-13": 0,
				"2010-05-14": 0,
				"2010-05-15": 0,
				"2010-05-16": 0,
				"2010-05-17": 0,
				"2010-05-18": 0,
				"2010-05-19": 0,
				"2010-05-20": 0,
				"2010-05-21": 0,
				"2010-05-22": 0,
				"2010-05-23": 0,
				"2010-05-24": 0,
				"2010-05-25": 0,
				"2010-05-26": 0,
				"2010-05-27": 0,
				"2010-05-28": 0,
				"2010-05-29": 0,
				"2010-05-30": 0,
				"2010-05-31": 0,
				"2010-06-01": 0,
				"2010-06-02": 0,
				"2010-06-03": 0,
				"2010-06-04": 0,
				"2010-06-05": 0,
				"2010-06-06": 0,
				"2010-06-07": 0,
				"2010-06-08": 0,
				"2010-06-09": 0,
				"2010-06-10": 0,
				"2010-06-11": 0,
				"2010-06-12": 0,
				"2010-06-13": 0,
				"2010-06-14": 0,
				"2010-06-15": 0,
				"2010-06-16": 0,
				"2010-06-17": 0,
				"2010-06-18": 0,
				"2010-06-19": 0,
				"2010-06-20": 0,
				"2010-06-21": 0,
				"2010-06-22": 0,
				"2010-06-23": 0,
				"2010-06-24": 0,
				"2010-06-25": 0,
				"2010-06-26": 0,
				"2010-06-27": 0,
				"2010-06-28": 0,
				"2010-06-29": 0,
				"2010-06-30": 0,
				"2010-07-01": 0,
				"2010-07-02": 0,
				"2010-07-03": 0,
				"2010-07-04": 0,
				"2010-07-05": 0,
				"2010-07-06": 0,
				"2010-07-07": 0,
				"2010-07-08": 0,
				"2010-07-09": 0,
				"2010-07-10": 0,
				"2010-07-11": 0,
				"2010-07-12": 0,
				"2010-07-13": 0,
				"2010-07-14": 0,
				"2010-07-15": 0,
				"2010-07-16": 0,
				"2010-07-17": 0,
				"2010-07-18": 0,
				"2010-07-19": 0,
				"2010-07-20": 0,
				"2010-07-21": 0,
				"2010-07-22": 0,
				"2010-07-23": 0,
				"2010-07-24": 0,
				"2010-07-25": 0,
				"2010-07-26": 0,
				"2010-07-27": 0,
				"2010-07-28": 0,
				"2010-07-29": 0,
				"2010-07-30": 0,
				"2010-07-31": 0,
				"2010-08-01": 0,
				"2010-08-02": 0,
				"2010-08-03": 0,
				"2010-08-04": 0,
				"2010-08-05": 0,
				"2010-08-06": 0,
				"2010-08-07": 0,
				"2010-08-08": 0,
				"2010-08-09": 0,
				"2010-08-10": 0,
				"2010-08-11": 0,
				"2010-08-12": 0,
				"2010-08-13": 0,
				"2010-08-14": 0,
				"2010-08-15": 0,
				"2010-08-16": 0,
				"2010-08-17": 0,
				"2010-08-18": 0,
				"2010-08-19": 0,
				"2010-08-20": 0,
				"2010-08-21": 0,
				"2010-08-22": 0,
				"2010-08-23": 0,
				"2010-08-24": 0,
				"2010-08-25": 0,
				"2010-08-26": 0,
				"2010-08-27": 0,
				"2010-08-28": 0,
				"2010-08-29": 0,
				"2010-08-30": 0,
				"2010-08-31": 0,
				"2010-09-01": 0,
				"2010-09-02": 0,
				"2010-09-03": 0,
				"2010-09-04": 0,
				"2010-09-05": 0,
				"2010-09-06": 0,
				"2010-09-07": 0,
				"2010-09-08": 0,
				"2010-09-09": 0,
				"2010-09-10": 0,
				"2010-09-11": 0,
				"2010-09-12": 0,
				"2010-09-13": 0,
				"2010-09-14": 0,
				"2010-09-15": 0,
				"2010-09-16": 0,
				"2010-09-17": 0,
				"2010-09-18": 0,
				"2010-09-19": 0,
				"2010-09-20": 0,
				"2010-09-21": 0,
				"2010-09-22": 0,
				"2010-09-23": 0,
				"2010-09-24": 0,
				"2010-09-25": 0,
				"2010-09-26": 0,
				"2010-09-27": 0,
				"2010-09-28": 0,
				"2010-09-29": 0,
				"2010-09-30": 0,
				"2010-10-01": 0,
				"2010-10-02": 0,
				"2010-10-03": 0,
				"2010-10-04": 0,
				"2010-10-05": 0,
				"2010-10-06": 0,
				"2010-10-07": 0,
				"2010-10-08": 0,
				"2010-10-09": 0,
				"2010-10-10": 0,
				"2010-10-11": 0,
				"2010-10-12": 0,
				"2010-10-13": 0,
				"2010-10-14": 0,
				"2010-10-15": 0,
				"2010-10-16": 0,
				"2010-10-17": 0,
				"2010-10-18": 0,
				"2010-10-19": 0,
				"2010-10-20": 0,
				"2010-10-21": 0,
				"2010-10-22": 0,
				"2010-10-23": 1,
				"2010-10-24": 2,
				"2010-10-25": 3,
				"2010-10-26": 4,
				"2010-10-27": 5,
				"2010-10-28": 6,
				"2010-10-29": 7,
				"2010-10-30": 8,
				"2010-10-31": 9,
				"2010-11-01": 10,
				"2010-11-02": 11,
				"2010-11-03": 12,
				"2010-11-04": 13,
				"2010-11-05": 14,
				"2010-11-06": 15,
				"2010-11-07": 16,
				"2010-11-08": 17,
				"2010-11-09": 18,
				"2010-11-10": 19,
				"2010-11-11": 20,
				"2010-11-12": 21,
				"2010-11-13": 22,
				"2010-11-14": 23,
				"2010-11-15": 24,
				"2010-11-16": 25,
				"2010-11-17": 26,
				"2010-11-18": 27,
				"2010-11-19": 28,
				"2010-11-20": 29,
				"2010-11-21": 30,
				"2010-11-22": 31,
				"2010-11-23": 32,
				"2010-11-24": 33,
				"2010-11-25": 34,
				"2010-11-26": 35,
				"2010-11-27": 36,
				"2010-11-28": 37,
				"2010-11-29": 38,
				"2010-11-30": 39,
				"2010-12-01": 40,
				"2010-12-02": 41,
				"2010-12-03": 42,
				"2010-12-04": 43,
				"2010-12-05": 44,
				"2010-12-06": 45,
				"2010-12-07": 46,
				"2010-12-08": 47,
				"2010-12-09": 48,
				"2010-12-10": 49,
				"2010-12-11": 50,
				"2010-12-12": 51,
				"2010-12-13": 52,
				"2010-12-14": 53,
				"2010-12-15": 54,
				"2010-12-16": 55,
				"2010-12-17": 56,
				"2010-12-18": 57,
				"2010-12-19": 58,
				"2010-12-20": 59,
				"2010-12-21": 60,
				"2010-12-22": 61,
				"2010-12-23": 62,
				"2010-12-24": 63,
				"2010-12-25": 64,
				"2010-12-26": 65,
				"2010-12-27": 66,
				"2010-12-28": 67,
				"2010-12-29": 68
			},
			"prev": {
				"total": 0
			}
		},
		"bandwidth": {
			"total": 38474400,
			"dates": {
				"2009-12-30": 0,
				"2009-12-31": 0,
				"2010-01-01": 0,
				"2010-01-02": 0,
				"2010-01-03": 0,
				"2010-01-04": 0,
				"2010-01-05": 0,
				"2010-01-06": 0,
				"2010-01-07": 0,
				"2010-01-08": 0,
				"2010-01-09": 0,
				"2010-01-10": 0,
				"2010-01-11": 0,
				"2010-01-12": 0,
				"2010-01-13": 0,
				"2010-01-14": 0,
				"2010-01-15": 0,
				"2010-01-16": 0,
				"2010-01-17": 0,
				"2010-01-18": 0,
				"2010-01-19": 0,
				"2010-01-20": 0,
				"2010-01-21": 0,
				"2010-01-22": 0,
				"2010-01-23": 0,
				"2010-01-24": 0,
				"2010-01-25": 0,
				"2010-01-26": 0,
				"2010-01-27": 0,
				"2010-01-28": 0,
				"2010-01-29": 0,
				"2010-01-30": 0,
				"2010-01-31": 0,
				"2010-02-01": 0,
				"2010-02-02": 0,
				"2010-02-03": 0,
				"2010-02-04": 0,
				"2010-02-05": 0,
				"2010-02-06": 0,
				"2010-02-07": 0,
				"2010-02-08": 0,
				"2010-02-09": 0,
				"2010-02-10": 0,
				"2010-02-11": 0,
				"2010-02-12": 0,
				"2010-02-13": 0,
				"2010-02-14": 0,
				"2010-02-15": 0,
				"2010-02-16": 0,
				"2010-02-17": 0,
				"2010-02-18": 0,
				"2010-02-19": 0,
				"2010-02-20": 0,
				"2010-02-21": 0,
				"2010-02-22": 0,
				"2010-02-23": 0,
				"2010-02-24": 0,
				"2010-02-25": 0,
				"2010-02-26": 0,
				"2010-02-27": 0,
				"2010-02-28": 0,
				"2010-03-01": 0,
				"2010-03-02": 0,
				"2010-03-03": 0,
				"2010-03-04": 0,
				"2010-03-05": 0,
				"2010-03-06": 0,
				"2010-03-07": 0,
				"2010-03-08": 0,
				"2010-03-09": 0,
				"2010-03-10": 0,
				"2010-03-11": 0,
				"2010-03-12": 0,
				"2010-03-13": 0,
				"2010-03-14": 0,
				"2010-03-15": 0,
				"2010-03-16": 0,
				"2010-03-17": 0,
				"2010-03-18": 0,
				"2010-03-19": 0,
				"2010-03-20": 0,
				"2010-03-21": 0,
				"2010-03-22": 0,
				"2010-03-23": 0,
				"2010-03-24": 0,
				"2010-03-25": 0,
				"2010-03-26": 0,
				"2010-03-27": 0,
				"2010-03-28": 0,
				"2010-03-29": 0,
				"2010-03-30": 0,
				"2010-03-31": 0,
				"2010-04-01": 0,
				"2010-04-02": 0,
				"2010-04-03": 0,
				"2010-04-04": 0,
				"2010-04-05": 0,
				"2010-04-06": 0,
				"2010-04-07": 0,
				"2010-04-08": 0,
				"2010-04-09": 0,
				"2010-04-10": 0,
				"2010-04-11": 0,
				"2010-04-12": 0,
				"2010-04-13": 0,
				"2010-04-14": 0,
				"2010-04-15": 0,
				"2010-04-16": 0,
				"2010-04-17": 0,
				"2010-04-18": 0,
				"2010-04-19": 0,
				"2010-04-20": 0,
				"2010-04-21": 0,
				"2010-04-22": 0,
				"2010-04-23": 0,
				"2010-04-24": 0,
				"2010-04-25": 0,
				"2010-04-26": 0,
				"2010-04-27": 0,
				"2010-04-28": 0,
				"2010-04-29": 0,
				"2010-04-30": 0,
				"2010-05-01": 0,
				"2010-05-02": 0,
				"2010-05-03": 0,
				"2010-05-04": 0,
				"2010-05-05": 0,
				"2010-05-06": 0,
				"2010-05-07": 0,
				"2010-05-08": 0,
				"2010-05-09": 0,
				"2010-05-10": 0,
				"2010-05-11": 0,
				"2010-05-12": 0,
				"2010-05-13": 0,
				"2010-05-14": 0,
				"2010-05-15": 0,
				"2010-05-16": 0,
				"2010-05-17": 0,
				"2010-05-18": 0,
				"2010-05-19": 0,
				"2010-05-20": 0,
				"2010-05-21": 0,
				"2010-05-22": 0,
				"2010-05-23": 0,
				"2010-05-24": 0,
				"2010-05-25": 0,
				"2010-05-26": 0,
				"2010-05-27": 0,
				"2010-05-28": 0,
				"2010-05-29": 0,
				"2010-05-30": 0,
				"2010-05-31": 0,
				"2010-06-01": 0,
				"2010-06-02": 0,
				"2010-06-03": 0,
				"2010-06-04": 0,
				"2010-06-05": 0,
				"2010-06-06": 0,
				"2010-06-07": 0,
				"2010-06-08": 0,
				"2010-06-09": 0,
				"2010-06-10": 0,
				"2010-06-11": 0,
				"2010-06-12": 0,
				"2010-06-13": 0,
				"2010-06-14": 0,
				"2010-06-15": 0,
				"2010-06-16": 0,
				"2010-06-17": 0,
				"2010-06-18": 0,
				"2010-06-19": 0,
				"2010-06-20": 0,
				"2010-06-21": 0,
				"2010-06-22": 0,
				"2010-06-23": 0,
				"2010-06-24": 0,
				"2010-06-25": 0,
				"2010-06-26": 0,
				"2010-06-27": 0,
				"2010-06-28": 0,
				"2010-06-29": 0,
				"2010-06-30": 0,
				"2010-07-01": 0,
				"2010-07-02": 0,
				"2010-07-03": 0,
				"2010-07-04": 0,
				"2010-07-05": 0,
				"2010-07-06": 0,
				"2010-07-07": 0,
				"2010-07-08": 0,
				"2010-07-09": 0,
				"2010-07-10": 0,
				"2010-07-11": 0,
				"2010-07-12": 0,
				"2010-07-13": 0,
				"2010-07-14": 0,
				"2010-07-15": 0,
				"2010-07-16": 0,
				"2010-07-17": 0,
				"2010-07-18": 0,
				"2010-07-19": 0,
				"2010-07-20": 0,
				"2010-07-21": 0,
				"2010-07-22": 0,
				"2010-07-23": 0,
				"2010-07-24": 0,
				"2010-07-25": 0,
				"2010-07-26": 0,
				"2010-07-27": 0,
				"2010-07-28": 0,
				"2010-07-29": 0,
				"2010-07-30": 0,
				"2010-07-31": 0,
				"2010-08-01": 0,
				"2010-08-02": 0,
				"2010-08-03": 0,
				"2010-08-04": 0,
				"2010-08-05": 0,
				"2010-08-06": 0,
				"2010-08-07": 0,
				"2010-08-08": 0,
				"2010-08-09": 0,
				"2010-08-10": 0,
				"2010-08-11": 0,
				"2010-08-12": 0,
				"2010-08-13": 0,
				"2010-08-14": 0,
				"2010-08-15": 0,
				"2010-08-16": 0,
				"2010-08-17": 0,
				"2010-08-18": 0,
				"2010-08-19": 0,
				"2010-08-20": 0,
				"2010-08-21": 0,
				"2010-08-22": 0,
				"2010-08-23": 0,
				"2010-08-24": 0,
				"2010-08-25": 0,
				"2010-08-26": 0,
				"2010-08-27": 0,
				"2010-08-28": 0,
				"2010-08-29": 0,
				"2010-08-30": 0,
				"2010-08-31": 0,
				"2010-09-01": 0,
				"2010-09-02": 0,
				"2010-09-03": 0,
				"2010-09-04": 0,
				"2010-09-05": 0,
				"2010-09-06": 0,
				"2010-09-07": 0,
				"2010-09-08": 0,
				"2010-09-09": 0,
				"2010-09-10": 0,
				"2010-09-11": 0,
				"2010-09-12": 0,
				"2010-09-13": 0,
				"2010-09-14": 0,
				"2010-09-15": 0,
				"2010-09-16": 0,
				"2010-09-17": 0,
				"2010-09-18": 0,
				"2010-09-19": 0,
				"2010-09-20": 0,
				"2010-09-21": 0,
				"2010-09-22": 0,
				"2010-09-23": 0,
				"2010-09-24": 0,
				"2010-09-25": 0,
				"2010-09-26": 0,
				"2010-09-27": 0,
				"2010-09-28": 0,
				"2010-09-29": 0,
				"2010-09-30": 0,
				"2010-10-01": 0,
				"2010-10-02": 0,
				"2010-10-03": 0,
				"2010-10-04": 0,
				"2010-10-05": 0,
				"2010-10-06": 0,
				"2010-10-07": 0,
				"2010-10-08": 0,
				"2010-10-09": 0,
				"2010-10-10": 0,
				"2010-10-11": 0,
				"2010-10-12": 0,
				"2010-10-13": 0,
				"2010-10-14": 0,
				"2010-10-15": 0,
				"2010-10-16": 0,
				"2010-10-17": 0,
				"2010-10-18": 0,
				"2010-10-19": 0,
				"2010-10-20": 0,
				"2010-10-21": 0,
				"2010-10-22": 0,
				"2010-10-23": 16400,
				"2010-10-24": 32800,
				"2010-10-25": 49200,
				"2010-10-26": 65600,
				"2010-10-27": 82000,
				"2010-10-28": 98400,
				"2010-10-29": 114800,
				"2010-10-30": 131200,
				"2010-10-31": 147600,
				"2010-11-01": 164000,
				"2010-11-02": 180400,
				"2010-11-03": 196800,
				"2010-11-04": 213200,
				"2010-11-05": 229600,
				"2010-11-06": 246000,
				"2010-11-07": 262400,
				"2010-11-08": 278800,
				"2010-11-09": 295200,
				"2010-11-10": 311600,
				"2010-11-11": 328000,
				"2010-11-12": 344400,
				"2010-11-13": 360800,
				"2010-11-14": 377200,
				"2010-11-15": 393600,
				"2010-11-16": 410000,
				"2010-11-17": 426400,
				"2010-11-18": 442800,
				"2010-11-19": 459200,
				"2010-11-20": 475600,
				"2010-11-21": 492000,
				"2010-11-22": 508400,
				"2010-11-23": 524800,
				"2010-11-24": 541200,
				"2010-11-25": 557600,
				"2010-11-26": 574000,
				"2010-11-27": 590400,
				"2010-11-28": 606800,
				"2010-11-29": 623200,
				"2010-11-30": 639600,
				"2010-12-01": 656000,
				"2010-12-02": 672400,
				"2010-12-03": 688800,
				"2010-12-04": 705200,
				"2010-12-05": 721600,
				"2010-12-06": 738000,
				"2010-12-07": 754400,
				"2010-12-08": 770800,
				"2010-12-09": 787200,
				"2010-12-10": 803600,
				"2010-12-11": 820000,
				"2010-12-12": 836400,
				"2010-12-13": 852800,
				"2010-12-14": 869200,
				"2010-12-15": 885600,
				"2010-12-16": 902000,
				"2010-12-17": 918400,
				"2010-12-18": 934800,
				"2010-12-19": 951200,
				"2010-12-20": 967600,
				"2010-12-21": 984000,
				"2010-12-22": 1000400,
				"2010-12-23": 1016800,
				"2010-12-24": 1033200,
				"2010-12-25": 1049600,
				"2010-12-26": 1066000,
				"2010-12-27": 1082400,
				"2010-12-28": 1098800,
				"2010-12-29": 1115200
			},
			"prev": {
				"total": 0
			}
		}
	}
}
